credentials/src/main/kotlin/web5/sdk/credentials/model/BitstringStatusListEntry.kt

```diff
@@ -0,0 +1,75 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import java.net.URI
+
+public const val DEFAULT_BITSTRING_STATUS_LIST_VC_TYPE: String = "BitstringStatusListCredential"
+public const val DEFAULT_BITSTRING_STATUS_LIST_ENTRY_TYPE: String = "BitstringStatusListEntry"
+public const val DEFAULT_STATUS_LIST_CONTEXT: String = "https://w3id.org/vc/status-list/2021/v1"
+private fun getObjectMapper(): ObjectMapper = jacksonObjectMapper().apply {
+  registerKotlinModule()
+  setSerializationInclusion(JsonInclude.Include.NON_NULL)
+}
+
+/**
+ * The [BitstringStatusListEntry] instance representing the core data model of a bitstring status list entry.
+ *
+ * @see {@link https://www.w3.org/TR/vc-bitstring-status-list/ | Bitstring Status List }
+ */
+public class BitstringStatusListEntry(
```

Comment:
is this class supposed to be a swap for CredentialStatus? i have not looked at danuubtech CredentialStatus, but the new name seems to indicate something a bit different than `somethingStatus`. could you explain the difference to me?

Reply:
yup its the updated spec, basically a renaming with a few things different

Reply:
Welp I was a bit wrong, reverting to Status List 2021

![image](https://github.com/TBD54566975/web5-kt/assets/5314059/7e77733d-e583-4b83-a6b8-00038c37f4db)